app/models/katello/repository.rb

```diff
@@ -495,6 +495,24 @@ def ostree_branch_names
       self.ostree_branches.map(&:name)
     end
 
+    def update_ostree_branches!(branch_names)
+      non_uniq = branch_names.select { |e| branch_names.count(e) > 1 }.uniq
```

Comment:
Can we avoid single letter variables in favor of something more descriptive? The name of the variable is 'non_uniq' even though you call 'uniq' at the end of the expression? Seems confusing.